demos/seasonal_contrast/pretrain.py

```diff
@@ -0,0 +1,92 @@
+#Pretraining models 
```

Comment:
Please add a comment with a disclaimer that this code is provided by Ankit and that we are only responsible for adding the tracking functions